Manifest.toml

```diff
@@ -0,0 +1,2 @@
+# This file is machine-generated - editing it directly is not advised
```

Comment:
For package purposes, do not commit `Manifest.toml` because this locks the version. Indeed, you should add `Manifest.toml` to `.gitignore` file